packages/vite-plugin/tests/out/vite-svelte/__snapshots__/serve.test.ts.snap

```diff
@@ -365,7 +365,7 @@ exports[`serve fs output > src/content.js-loader.js 1`] = `
 `;
 
 exports[`serve fs output > src/lib/Counter.svelte.js 1`] = `
-"import { createHotContext as __vite__createHotContext } from \\"/vendor/vite-client.js\\";import.meta.hot = __vite__createHotContext(\\"/src/lib/Counter.svelte.js\\");/* src/lib/Counter.svelte generated by Svelte v3.48.0 */
+"import { createHotContext as __vite__createHotContext } from \\"/vendor/vite-client.js\\";import.meta.hot = __vite__createHotContext(\\"/src/lib/Counter.svelte.js\\");/* src/lib/Counter.svelte generated by Svelte v3.59.2 */
```

Comment:
how is that different? no `package-lock.json` or not running `npm ci`?

Reply:
same issue as https://github.com/crxjs/chrome-extension-tools/pull/997#discussion_r2030621822 - fixed with latest force-push